check_borg

```diff
@@ -118,13 +119,30 @@ case "$(${BORG} --version | cut -d' ' -f2)" in
 		;;
 	*)
 		last="$(${BORG} list --sort timestamp --last 1 --format '{time}')"
-		[ "$?" = 0 ] || error "Cannot list repository archives. Repo Locked?"
+		if [ "$?" = 0 ]
+		then
+			if not ps aux | grep "${BORG}" | grep 'create' | grep "$BORG_REPO" >> /dev/null
```

Comment:
You may want to grep for the exact order by using `grep -E 'PATTERN1.*PATTERN2'` or by using `awk '/PATTERN1.*PATTERN2/'` or `sed '/PATTERN1.*PATTERN2/!d'`.